app/admin/issuances.rb

```diff
@@ -0,0 +1,87 @@
+ActiveAdmin.register Issuance do
+  #actions :all, except: [:edit, :destroy]
+
+  # See permitted parameters documentation:
+  # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+  #
+  # Uncomment all parameters which should be permitted for assignment
+  #
+  permit_params :status, :gift_card_amount, :card_amount, :quantity, :begin_use_date, :end_use_date, :expiration_date, :gift_card_type_id
+  #
+  # or
+  #
+  # permit_params do
+  #   permitted = [:status, :initiator_id, :card_amount, :quantity, :begin_use_date, :end_use_date, :expiration_date]
+  #   permitted << :other if params[:action] == 'create' && current_user.admin?
+  #   permitted
+  # end
+  
+  index do
+    id_column
+    tag_column :status
+    column :created_at
+    column :gift_card_type
+    column :creator
+    column :issuer
+    number_column :card_amount, as: :currency, unit: "$"
+    column :quantity
+    column :min_certificate do |issuance|
+      issuance.allocated_certificates.split(", ").first
```

Comment:
This is safe because only the code generates `allocated_certificates`, via `.join(", ")`, correct?

Still, it would be best to set a global constant for `", "`, and use that throughout.

I would also advocate for dropping the space after the comma.